test/src/e2e_vm_tests/test_programs/should_fail/excess_fn_arguments/test.toml

```diff
@@ -1 +1,5 @@
-category = "fail"
+# This test should be a "fail" but it's currently passing.
```

Comment:
I wonder we should have a flag or something that indicates that the test is known not to pass right now but _should_ in the future? Then we could potentially print a list of known not-yet-passing tests at the end of the test output as a reminder? That said I guess having an issue for these is probably enough. Either way, not a concern of this PR!

Reply:
Ah I just noticed we have a `disabled` category - guess we could use that?

Reply:
Yar, it prints the number of disabled tests at the end, I guess we could keep an eye on that.

Having an actual GitHub issue for things that should change in the future is a bit more robust.  I reckon having a flag in the suite would just get forgotten too easily.